### PR TITLE
Relaxr CSS 

### PR DESCRIPTION
**Pull request by designsbydali**

### CODE REVIEW

File: starter_code/css/style.css [hunk 1]
```diff
@@ -10,3 +10,97 @@ Developer @GA in Austin, TX
 /* Custom Styles
 /*******************************************/
 
+body {
+  font-family: Open Sans;
+}
+
+header {
+  background-image: url(../images/header_bg.jpg);
```
Comment (rldaggie) [inline starter_code/css/style.css:18]: Nice job using relative paths

File: starter_code/css/style.css [hunk 2]
```diff
@@ -10,3 +10,97 @@ Developer @GA in Austin, TX
...
+  background-size: cover;
+  padding: 50px 0;
+  text-align: center;
+}
+h1.logo {
+  /*position: relative;*/
+  font-weight: 800;
+  font-size: 48px;
+  text-align: left;
+  color: #ffffff;
```
Comment (rldaggie) [inline starter_code/css/style.css:29]: You could probably put `color: #ffffff` on `header` and then both the `h1` and `h2` will be white

File: starter_code/css/style.css [hunk 3]
```diff
@@ -10,3 +10,97 @@ Developer @GA in Austin, TX
...
+  padding:0px 0px 0px 170px;
+}
+h2 {
+  position: relative;
+  font-size: 32px;
+  font-weight: 600;
+  color: #ffffff;
+  margin-bottom: 250px;
+  /*padding: 0px 0px 350px 450px;*/
```
Comment (rldaggie) [inline starter_code/css/style.css:38]: If you no longer need this I'd delete it

**Comment by rldaggie** [comment]
@designsbydali great job! Your css is looking really good. The main feedback I'd have is to put the primary `color` styles on the container element, rather than having to use it multiple times on child elements. This will keep your code DRY

**Comment by designsbydali** [comment]
got it thanks a lot Riley!

**Pull request closed without merging**